src/content/quickstart/kind/_index.md

```diff
@@ -52,28 +53,20 @@ subctl join --kubeconfig output/kubeconfigs/kind-config-cluster3 broker-info.sub
 
 You now have a Submariner environment that you can experiment with.
 
-### Deploying a Basic Environment
 
-If you wish to deploy just a basic multi cluster environment, run:
+####  Verify Deployment
+To manually verify the deployment follow the steps below.
 
 ```bash
-git clone https://github.com/submariner-io/shipyard
-cd shipyard
-make deploy
+kubectl --kubeconfig output/kubeconfigs/kind-config-cluster3 create deployment nginx --image=nginx
+kubectl --kubeconfig output/kubeconfigs/kind-config-cluster3 expose deployment nginx --port=80
+kubectl --kubeconfig output/kubeconfigs/kind-config-cluster2 run --generator=run-pod/v1 tmp-shell --rm -i --tty --image quay.io/submariner/nettest -- /bin/bash
+curl nginx
 ```
 
-At the end of the deployment you'll have a very basic environment that you can experiment with.
-
-### Deploying Like a Submariner
-
-To deploy the environment like Submariner does it, run:
+You can also verify manually with the help of `subctl verify-connectivity`
```

Comment:
As running "subctl verify-connectivity" is not a **manual** test, can you please rephrase this sentence as follows.

Alternately, you can also run the Submariner test-suite that validates various use-cases:

Reply:
:+1: